.github/workflows/linux.yml

```diff
@@ -0,0 +1,19 @@
+name: Build Linux
+on:
```

Comment:
Lets make them run on Push to PR and Master as well

Reply:
Done